dbt/adapters/spark_cde/cdeapisession.py

```diff
@@ -738,6 +740,29 @@ def connect(self, user_name, password, base_auth_url, base_api_url, session_para
             "accept": "text/plain; charset=utf-8",
         }
 
+        # check for dbt-spark-cde specific params
+        dbt_cde_session_param_keys = ["dbt.cde.job_timeout", "dbt.cde.poll_wait", "dbt.cde.log_wait"]
+
+        try:
+          new_session_params = {}
+          for key, value in self.session_params.items():
+            if key in dbt_cde_session_param_keys:
+               if key == "dbt.cde.job_timeout":
+                  DEFAULT_CDE_JOB_TIMEOUT = int(value)
+               elif key == "dbt.cde.poll_wait":
+                  DEFAULT_POLL_WAIT = int(value)
+               elif key == "dbt.cde.log_wait":
+                  DEFAULT_LOG_WAIT = int(value)
+            else:
+               new_session_params[key] = value
+
+          # remove adapter specific params
+          self.session_params = new_session_params
+        except Exception as e:
+          logger.info(f"Unable to set dbt-spark-cde parameters: {e}")
+
+        logger.debug(f"[spark-cde config] timeout = {DEFAULT_CDE_JOB_TIMEOUT}, poll_wait = {DEFAULT_POLL_WAIT}, log_wait = {DEFAULT_LOG_WAIT}")
```

Comment:
Please just mention in debug that it is in seconds.

Reply:
@sujitkp-blr added this.